Casks/c/chordpotion.rb

```diff
@@ -0,0 +1,32 @@
+cask "chordpotion" do
+  version "2.4.2"
+  sha256 "c97c8a34d92ff08ff1cbf959d40b486c1e60b5cadd081f53fda590d57d585be9"
+
+  url "https://fys.ams3.cdn.digitaloceanspaces.com/data/ChordPotion_#{version}_mac.zip",
+      verified: "fys.ams3.cdn.digitaloceanspaces.com/data/"
+  name "ChordPotion"
+  desc "MIDI plug-in to transform chords into riffs and melodies"
+  homepage "https://feelyoursound.com/chordpotion"
```

Comment:
```suggestion
  homepage "https://feelyoursound.com/chordpotion/"
```

Let's resolve the redirect.